.changeset/brave-pots-drop.md

```diff
@@ -0,0 +1,18 @@
+---
+'astro': minor
+---
+
+Partials
```

Comment:
```suggestion
Page Partials
```

or Partial Pages? or just Partials! Decide how you'd like to colloquially refer to these. I think anything is fine, just want to make sure you've made a conscious choice.